server/utils/applications/utils.ts

```diff
@@ -74,8 +74,11 @@ const createNameAnchorElement = (name: string, applicationId: string) => {
   )
 }
 
-export const createWithdrawAnchorElement = (applicationId: string) => {
-  return htmlValue(`<a href=${paths.applications.withdraw.confirm({ id: applicationId })}>Withdraw</a>`)
+export const createWithdrawElement = (applicationId: string, application: ApplicationSummary) => {
+  if (application?.submittedAt)
+    return htmlValue(`<a href=${paths.applications.withdraw.confirm({ id: applicationId })}>Withdraw</a>`)
+
+  return textValue('Cannot withdraw')
```

Comment:
I feel like returning nothing if we can't withdraw, with the table heading as "Actions" might be better here

Reply:
Makes sense to me